### PR TITLE
initial feature

### PR DESCRIPTION
**Pull request by brettdarco**
Co-authored-by: Dieter Wellerdiek <wellerdiek@de.ibm.com>
Co-authored-by: Tobias Huschle <tobias.huschle@de.ibm.com>
Co-authored-by: Toni Pohl <info@toni-pohl.de>
Co-authored-by: Marcos Barbieri <marcos.barbieri@ibm.com>

Signed-off-by: Dieter Wellerdiek <wellerdiek@de.ibm.com>

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""Build information for `pysvc` project
```
Comment (Dieter-W) [inline setup.py]: What's pysvc?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,53 @@
+#
+
+.PHONY: clean
+clean: ## clean project
+clean:
```
Comment (Dieter-W) [inline Makefile:8]: There are several cases where we have two label like here, is that correct? 
```
clean:  ## clean project
clean: 
```

File: setup.py (hunk 1 above)
Comment (brettdarco) [reply setup.py]: Copy paste error fixed.

File: Makefile (hunk 2 above)
Comment (marcosb1) [reply Makefile:8]: That is by design, it is for the `help` target to parse the description of each make target so taht it displays when the user runs `make help`

**Comment by brettdarco** [comment]
@Dieter-W @marcosb1 
Added copyright text. Need review.

File: Makefile (hunk 2 above)
Comment (Dieter-W) [reply Makefile:8]: Thanks

File: setup.py (hunk 1 above)
Comment (Dieter-W) [reply setup.py]: thanks

**Review by Dieter-W — APPROVED** [review]

**Pull request merged**